inc/cleantalk-public-integrations.php

```diff
@@ -830,8 +834,12 @@ function apbct_form__formidable__testSpam($errors, $_form)
         array(
             'message'         => $message,
             'sender_email'    => $sender_email,
+            'sender_emails_array' => $sender_emails_array,
```

Comment:
Please, remove this parameter `sender_emails_array`

Reply:
ok